packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -254,6 +256,18 @@ export const getCanCreateChat = createSelector(
       }
     }
 
+    // Check for existing chat and short circuit if exists
+    // Note: this only works if the respective chat has been fetched already
```

Comment:
This prevents the new user list from claiming you can't create a chat with someone, since you're not "creating" a chat if you already have one